example/src/NativeBannerExample.tsx

```diff
@@ -21,6 +22,10 @@ const NativeBannerExample = ({
     isProgrammaticBannerShowing,
     setIsNativeUIBannerShowing,
 }: Props) => {
+    const adaptiveBannerEnabled = true; // true by default
```

Comment:
Our plugin should automatically have adaptive banners enabled by default already though, so do we need this?

Reply:
Removed.